dask/dataframe/io/parquet/core.py

```diff
@@ -796,14 +816,20 @@ def get_engine(engine):
         _ENGINES["fastparquet"] = eng = FastParquetEngine
         return eng
 
-    elif engine == "pyarrow" or engine == "arrow":
+    elif engine in ("pyarrow", "arrow", "pyarrow-legacy", "pyarrow-dataset"):
```

Comment:
Oh here we go. Can you mention `pyarrow-legacy` in the docstring?

Reply:
Will do - Sorry for leaving that out